### ISSUE TITLE
GPU memory

### ISSUE THREAD

**Issue by kaigelee** [CSIPlab/EBAD#2]
Great job.
I would like to ask what hardware environment is used in the paper? Such as RTX 3090?
And what is the minimum required GPU memory?

**Comment by zikuicai** [comment]
Yes, we used a RTX 3090 GPU, and detailed runtime and resource usage information can be found in the supplementary material. The memory demand varies based on the ensemble size and individual model size. For small ensembles, it is possible to fit them within GPUs with lower memory capacities, such as the 2080 Ti.
